emctl/.goreleaser.yml

```diff
@@ -0,0 +1,41 @@
+before:
+  hooks:
+    # You may remove this if you don't use go modules.
+    - go mod tidy
+    # you may remove this if you don't need go generate
+    - go generate ./...
+
+snapshot:
+  name_template: '{{ .Tag }}'
+checksum:
+  name_template: 'checksums.txt'
+changelog:
+  skip: true
+
+builds:
+  - id: emctl
+    main: cmd/client/main.go
+    binary: bin/egctl
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - darwin
+    ldflags:
+      - -s -w
+      - -X github.com/megaease/easemesh/emctl/pkg/version.RELEASE=v1.0.0
```

Comment:
always v1.0.0?

Reply:
Of course not, but we need to update it in every release. I did the research for it in Easegress before, the goreleaser doesn't support template here..